canvasapi/canvas_object.py

```diff
@@ -77,5 +87,14 @@ def to_json(self):
         """
         Return the original JSON response from the API that was used to
         construct the object.
+
+        .. warning::
+            .. deprecated:: 0.14.0
+                Use :func:`canvasapi.assignment.Assignment.get_submission` instead.
         """
+        warnings.warn(
+            "`CanvasObject.to_json()` is deprecated and will be removed in a future version. "
+            "To see the original responses, check Requester._cache",
```

Comment:
Should we tell them to enable request logging instead?

Reply:
That sounds good! Could you please provide a better warning string?

Reply:
```
warnings.warn(
 "`CanvasObject.to_json()` is deprecated and will be removed in a future version. "
 "To view the original attributes sent by Canvas, enable logs from the requests library",
)
```